### ISSUE TITLE
Build failed on Debian

### ISSUE THREAD

**Issue by pavroo** [opentoonz/opentoonz#851]
Hi
I tried to build the opentoonz on Debian testing 64 bit installation.
It failed (twice), the second time with "verbos" mode:
`[100%] Linking CXX executable ../bin/OpenToonz_1.1
cd /home/builds/opentoonz/64/opentoonz-1.1.0/toonz/build/toonz && /usr/bin/cmake -E cmake_link_script CMakeFiles/OpenToonz_1.1.dir/link.txt --verbose=1
/usr/bin/c++    -Wwrite-strings -Wundef -Wmissing-declarations  -fpermissive -lstdc++ -std=c++11 -O3 -DNDEBUG   CMakeFiles/OpenToonz_1.1.dir/floatingpanelcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/canvassizepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/history.cpp.o CMakeFiles/OpenToonz_1.1.dir/loadfoldercommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/loadfolderpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/main.cpp.o CMakeFiles/OpenToonz_1.1.dir/mainwindow.cpp.o CMakeFiles/OpenToonz_1.1.dir/matchline.cpp.o CMakeFiles/OpenToonz_1.1.dir/menubar.cpp.o CMakeFiles/OpenToonz_1.1.dir/menubarpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/mergecolumns.cpp.o CMakeFiles/OpenToonz_1.1.dir/moviegenerator.cpp.o CMakeFiles/OpenToonz_1.1.dir/pane.cpp.o CMakeFiles/OpenToonz_1.1.dir/previewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/previewfxmanager.cpp.o CMakeFiles/OpenToonz_1.1.dir/ruler.cpp.o CMakeFiles/OpenToonz_1.1.dir/scanlist.cpp.o CMakeFiles/OpenToonz_1.1.dir/tapp.cpp.o CMakeFiles/OpenToonz_1.1.dir/toolbar.cpp.o CMakeFiles/OpenToonz_1.1.dir/batchserversviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/castviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/exportlevelpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/exportscenepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/colormodelviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/exportpanel.cpp.o CMakeFiles/OpenToonz_1.1.dir/filebrowser.cpp.o CMakeFiles/OpenToonz_1.1.dir/filebrowserversioncontrol.cpp.o CMakeFiles/OpenToonz_1.1.dir/filmstrip.cpp.o CMakeFiles/OpenToonz_1.1.dir/matchlinecommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/messagepanel.cpp.o CMakeFiles/OpenToonz_1.1.dir/tasksviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/testpanel.cpp.o CMakeFiles/OpenToonz_1.1.dir/tpanels.cpp.o CMakeFiles/OpenToonz_1.1.dir/xshcellmover.cpp.o CMakeFiles/OpenToonz_1.1.dir/xshcellviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/xshcolumnviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/xsheetdragtool.cpp.o CMakeFiles/OpenToonz_1.1.dir/xsheetviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/xshnoteviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/xshrowviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/sceneviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/sceneviewerevents.cpp.o CMakeFiles/OpenToonz_1.1.dir/viewerdraw.cpp.o CMakeFiles/OpenToonz_1.1.dir/viewerpane.cpp.o CMakeFiles/OpenToonz_1.1.dir/castselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/cellselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/columnselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/fileselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/filmstripselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/keyframeselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/addfilmstripframespopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/camerasettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/convertpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/duplicatepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/dvdirtreeview.cpp.o CMakeFiles/OpenToonz_1.1.dir/filebrowserpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/formatsettingspopups.cpp.o CMakeFiles/OpenToonz_1.1.dir/fxparameditorpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/histogrampopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/insertfxpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/levelcreatepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/levelsettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/linetestcapturepane.cpp.o CMakeFiles/OpenToonz_1.1.dir/linetestpane.cpp.o CMakeFiles/OpenToonz_1.1.dir/linetestviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/magpiefileimportpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/outputsettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/overwritepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/pltgizmopopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/preferencespopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/projectpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/psdsettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/renumberpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/runscriptcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/savepresetpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/scanpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/sceneviewercontextmenu.cpp.o CMakeFiles/OpenToonz_1.1.dir/scenesettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/scriptconsolepanel.cpp.o CMakeFiles/OpenToonz_1.1.dir/shifttracetool.cpp.o CMakeFiles/OpenToonz_1.1.dir/shortcutpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/subcameramanager.cpp.o CMakeFiles/OpenToonz_1.1.dir/timestretchpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/trackerpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/vectorizerpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/vectorizerswatch.cpp.o CMakeFiles/OpenToonz_1.1.dir/versioncontrol.cpp.o CMakeFiles/OpenToonz_1.1.dir/versioncontroltimeline.cpp.o CMakeFiles/OpenToonz_1.1.dir/versioncontrolxmlreader.cpp.o CMakeFiles/OpenToonz_1.1.dir/versioncontrolwidget.cpp.o CMakeFiles/OpenToonz_1.1.dir/viewerpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/cellselectioncommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/columncommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/filmstripcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/iocommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/levelcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/print.cpp.o CMakeFiles/OpenToonz_1.1.dir/subscenecommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/svncleanupdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/svncommitdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/svndeletedialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/svnlockdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/svnlockframerangedialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/svnrevertdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/svnupdateandlockdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/svnupdatedialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/svnpurgedialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/vcrcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/xsheetcmd.cpp.o CMakeFiles/OpenToonz_1.1.dir/celldata.cpp.o CMakeFiles/OpenToonz_1.1.dir/drawingdata.cpp.o CMakeFiles/OpenToonz_1.1.dir/filedata.cpp.o CMakeFiles/OpenToonz_1.1.dir/imageviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/keyframedata.cpp.o CMakeFiles/OpenToonz_1.1.dir/dvitemview.cpp.o CMakeFiles/OpenToonz_1.1.dir/dvwidgets.cpp.o CMakeFiles/OpenToonz_1.1.dir/flipbook.cpp.o CMakeFiles/OpenToonz_1.1.dir/frameheadgadget.cpp.o CMakeFiles/OpenToonz_1.1.dir/onionskinmaskgui.cpp.o CMakeFiles/OpenToonz_1.1.dir/batches.cpp.o CMakeFiles/OpenToonz_1.1.dir/tfarmstuff.cpp.o CMakeFiles/OpenToonz_1.1.dir/cleanuppaletteviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/cleanuppopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/cleanuppreview.cpp.o CMakeFiles/OpenToonz_1.1.dir/cleanupsettingsmodel.cpp.o CMakeFiles/OpenToonz_1.1.dir/cleanupsettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/cleanupswatch.cpp.o CMakeFiles/OpenToonz_1.1.dir/cellkeyframedata.cpp.o CMakeFiles/OpenToonz_1.1.dir/cellkeyframeselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/keyframemover.cpp.o CMakeFiles/OpenToonz_1.1.dir/cachefxcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/exportlevelcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/adjustlevelspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/adjustthicknesspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/binarizepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/brightnessandcontrastpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/linesfadepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/rendercommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/curveio.cpp.o CMakeFiles/OpenToonz_1.1.dir/meshifypopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/selectionutils.cpp.o CMakeFiles/OpenToonz_1.1.dir/antialiaspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/mergecmapped.cpp.o CMakeFiles/OpenToonz_1.1.dir/comboviewerpane.cpp.o CMakeFiles/OpenToonz_1.1.dir/historypane.cpp.o CMakeFiles/OpenToonz_1.1.dir/cleanupsettingspane.cpp.o CMakeFiles/OpenToonz_1.1.dir/penciltestpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/dummyprocessor.cpp.o CMakeFiles/OpenToonz_1.1.dir/metnum.cpp.o CMakeFiles/OpenToonz_1.1.dir/ObjectTracker.cpp.o CMakeFiles/OpenToonz_1.1.dir/predict3d.cpp.o CMakeFiles/OpenToonz_1.1.dir/qrc_toonz.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_addfilmstripframespopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_adjustlevelspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_adjustthicknesspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_antialiaspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_batches.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_batchserversviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_binarizepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_brightnessandcontrastpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cachefxcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_camerasettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_canvassizepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_castselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_castviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_celldata.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cellkeyframedata.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cellkeyframeselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cellselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cleanuppaletteviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cleanuppopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cleanuppreview.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cleanupsettingsmodel.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cleanupsettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cleanupswatch.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_colormodelviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_columncommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_columnselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_convertpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_curveio.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_drawingdata.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_duplicatepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_dvdirtreeview.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_dvitemview.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_dvwidgets.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_exportlevelcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_exportlevelpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_exportscenepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_exportpanel.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_filebrowser.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_filebrowsermodel.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_filebrowserpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_filedata.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_fileinfopopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_fileselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_filmstrip.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_filmstripcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_filmstripselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_flipbook.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_floatingpanelcommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_formatsettingspopups.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_frameheadgadget.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_fxparameditorpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_histogrampopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_history.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_imageviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_insertfxpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_iocommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_keyframedata.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_keyframeselection.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_keyframemover.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_levelcreatepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_levelsettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_linesfadepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_linetestcapturepane.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_linetestpane.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_linetestviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_loadfolderpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_magpiefileimportpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_mainwindow.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_matchline.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_menubar.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_menubarpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_menubarcommandids.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_meshifypopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_messagepanel.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_moviegenerator.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_onionskinmaskgui.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_outputsettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_overwritepopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_pane.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_pltgizmopopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_preferencespopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_previewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_previewfxmanager.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_projectpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_psdsettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_renumberpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_reslist.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_ruler.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_savepresetpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_scanlist.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_scanpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_scenesettingspopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_sceneviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_sceneviewercontextmenu.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_sceneviewerevents.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_scriptconsolepanel.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_selectionutils.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_shifttracetool.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_shortcutpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_subcameramanager.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_subscenecommand.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svncleanupdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svncommitdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svndeletedialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svnlockdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svnlockframerangedialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svnrevertdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svnupdateandlockdialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svnupdatedialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_svnpurgedialog.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_tapp.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_tasksviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_testpanel.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_tfarmstuff.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_timestretchpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_toolbar.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_tpanels.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_trackerpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_vectorizerpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_vectorizerswatch.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_versioncontrol.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_versioncontrolgui.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_versioncontroltimeline.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_versioncontrolxmlreader.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_versioncontrolwidget.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_viewerdraw.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_viewerpane.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_viewerpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_xshcellmover.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_xshcellviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_xshcolumnviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_xsheetdragtool.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_xsheetviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_xshnoteviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_xshrowviewer.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_comboviewerpane.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_historypane.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_cleanupsettingspane.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_penciltestpopup.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_dummyprocessor.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_metnum.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_ObjectTracker.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_predict3d.cpp.o CMakeFiles/OpenToonz_1.1.dir/moc_processor.cpp.o CMakeFiles/OpenToonz_1.1.dir/filebrowsermodel.cpp.o  -o ../bin/OpenToonz_1.1  -L/home/builds/opentoonz/64/opentoonz-1.1.0/toonz/build/lib/opentoonz -rdynamic /usr/lib/x86_64-linux-gnu/libQt5Svg.so.5.6.1 /usr/lib/x86_64-linux-gnu/libQt5Xml.so.5.6.1 /usr/lib/x86_64-linux-gnu/libQt5PrintSupport.so.5.6.1 /usr/lib/x86_64-linux-gnu/libQt5Multimedia.so.5.6.1 -lGL -lglut -lGLU ../lib/opentoonz/libcolorfx.so ../lib/opentoonz/libimage.so ../lib/opentoonz/libtnztools.so -ltnzstdfx -ltfarm -lblas /home/builds/opentoonz/64/opentoonz-1.1.0/thirdparty/tiff-4.0.3/libtiff/.libs/libtiff.a -llzma -lpng ../lib/opentoonz/libtoonzqt.so ../lib/opentoonz/libtoonzlib.so /usr/lib/x86_64-linux-gnu/libQt5Script.so.5.6.1 ../lib/opentoonz/libtnzext.so ../lib/opentoonz/libtnzbase.so -lSDL2 -lsuperlu ../lib/opentoonz/libsound.so ../lib/opentoonz/libtnzcore.so /usr/lib/x86_64-linux-gnu/libQt5Network.so.5.6.1 /usr/lib/x86_64-linux-gnu/libQt5OpenGL.so.5.6.1 /usr/lib/x86_64-linux-gnu/libQt5Widgets.so.5.6.1 /usr/lib/x86_64-linux-gnu/libQt5Gui.so.5.6.1 /usr/lib/x86_64-linux-gnu/libQt5Core.so.5.6.1 -lGL -lglut -lz -ljpeg -llz4 -lGLU -lblas -Wl,-rpath,/home/builds/opentoonz/64/opentoonz-1.1.0/toonz/build/lib/opentoonz: 
../lib/opentoonz/libimage.so: undefined reference to`jbg_dec_getsize'
../lib/opentoonz/libimage.so: undefined reference to `jbg_dec_init'
../lib/opentoonz/libimage.so: undefined reference to`jbg_dec_free'
../lib/opentoonz/libimage.so: undefined reference to `jbg_newlen'
../lib/opentoonz/libimage.so: undefined reference to`jbg_strerror'
../lib/opentoonz/libimage.so: undefined reference to `jbg_enc_init'
../lib/opentoonz/libimage.so: undefined reference to`jbg_enc_out'
../lib/opentoonz/libimage.so: undefined reference to `jbg_enc_free'
../lib/opentoonz/libimage.so: undefined reference to`jbg_dec_getimage'
../lib/opentoonz/libimage.so: undefined reference to `jbg_dec_in'
collect2: error: ld returned 1 exit status
toonz/CMakeFiles/OpenToonz_1.1.dir/build.make:8846: recipe for target 'bin/OpenToonz_1.1' failed
make[2]: *** [bin/OpenToonz_1.1] Error 1
make[2]: Leaving directory '/home/builds/opentoonz/64/opentoonz-1.1.0/toonz/build'
CMakeFiles/Makefile2:1014: recipe for target 'toonz/CMakeFiles/OpenToonz_1.1.dir/all' failed
make[1]: *** [toonz/CMakeFiles/OpenToonz_1.1.dir/all] Error 2
make[1]: Leaving directory '/home/builds/opentoonz/64/opentoonz-1.1.0/toonz/build'
Makefile:127: recipe for target 'all' failed
make: *** [all] Error 2
`

**Comment by ideasman42** [comment]
Looks to be libtiff related, did you build the local libtiff? *(as noted in the instructions)*

**Comment by pavroo** [comment]
Yes i did, I followed the doc page for Linux, step by step.

**Comment by caryoscelus** [comment]
Now that the previous build issue is fixed, i get the same error as here.

```
../lib/opentoonz/libimage.so: undefined reference to `jbg_dec_getsize'
../lib/opentoonz/libimage.so: undefined reference to `jbg_dec_init'
../lib/opentoonz/libimage.so: undefined reference to `jbg_dec_free'
../lib/opentoonz/libimage.so: undefined reference to `jbg_newlen'
../lib/opentoonz/libimage.so: undefined reference to `jbg_strerror'
../lib/opentoonz/libimage.so: undefined reference to `jbg_enc_init'
../lib/opentoonz/libimage.so: undefined reference to `jbg_enc_out'
../lib/opentoonz/libimage.so: undefined reference to `jbg_enc_free'
../lib/opentoonz/libimage.so: undefined reference to `jbg_dec_getimage'
../lib/opentoonz/libimage.so: undefined reference to `jbg_dec_in'
```

Configuration says it uses correct libtiff:

```
***** libtiff Header path:/data/all/Downloads/Apps/opentoonz/thirdparty/tiff-4.0.3/libtiff
***** libtiff Library path:/data/all/Downloads/Apps/opentoonz/thirdparty/tiff-4.0.3/libtiff/.libs/libtiff.a
```

And inspecting libtiff.a results in
```
opentoonz/thirdparty/tiff-4.0.3/libtiff/.libs$ nm libtiff.a  | grep jbg
                 U jbg_dec_free
                 U jbg_dec_getimage
                 U jbg_dec_getsize
                 U jbg_dec_in
                 U jbg_dec_init
                 U jbg_enc_free
                 U jbg_enc_init
                 U jbg_enc_out
                 U jbg_newlen
                 U jbg_strerror
```

**Comment by caryoscelus** [comment]
Apparently this can be fixed by passing "--disable-jbig" to libtiff configure and recompiling.

**Comment by ideasman42** [comment]
Just built on Ubuntu 16.10 amd64 without any problems, not sure why it might fail.

You ran this command in libtiff directory?

     CFLAGS="-fPIC" ./configure && make

Further, can only suggest to run `make VERBOSE=1` then manually copy and paste the commands that fail into a shell script, making manual adjustments to see if you can resolve the linking issue to find the cause.

Also, the customizations to libtiff are minor, opentoonz could be made to use the systems libtiff, probably end up being less of a hassle.
Or the changes could be submitted to `libtiff` project to include.

**Comment by j3pic** [comment]
The cause of this problem has several parts to it:

1. OpenToonz includes its own libtiff instead of using the system version. It's too hard to get OpenToonz and its CMake scripts to generate Makefiles that use the system version of libtiff, so I just compiled the included version with the default options. But that results in a libtiff that depends on `-ljbig` if that library is detected by the configure script. I'd expect that you'd avoid the error if you build on a system that doesn't have `libjbig-dev` installed, which would lead to difficulty in reproducing the issue.
2. OpenToonz doesn't include `-ljbig` in link commands involving `-ltiff`, even though it does include libtiff's other dependencies. I don't know whether this is due to hard-coding the link command or some other issue. Using `pkg-config libtiff-4 --libs --static` to determine the linker flags for libtiff would've avoided this problem.
3. Why is this closed? The problem was never solved. I cloned this repo *today* and got the error described here.
